recipe/meta.yaml

```diff
@@ -9,7 +8,7 @@ package:
 source:
   # don't use github generated tar-ball as the sha256 value may not be stable
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: c910b8fbfd67b8e9d0b10ebc012b23cd67cbecef1b96f00d391ddd298d71671c
```

Comment:
Apparently, a patch is needed for aarch64:
https://github.com/numba/llvmlite/blob/v0.40.0/CHANGE_LOG#L8
https://github.com/numba/llvmlite/pull/926

Reply:
this patch will be applied on llvmdev-feedstock